src/path_info.cpp

```diff
@@ -104,6 +104,10 @@ void PATH_INFO::set_standard_filenames()
     // Special: data_dir and gfx_dir
     std::string prefix;
     cata_path prefix_path;
+
+    // Data is always relative to itself. Also, the base path might not be writeable.
+    datadir_path_value = cata_path{ cata_path::root_path::data, fs::path{} };
+
     if( !base_path_value.empty() ) {
 #if defined(DATA_DIR_PREFIX)
```

Comment:
Yeah the problem happens inside this block at line 111 (before code). I didn't notice that prefix_path was reading from `datadir_path_value` which clearly isn't initialized yet. Then `prefix_path_value` is forever bad and is what causes issues down at line 127.